docs/README.md

```diff
@@ -48,6 +48,35 @@ Then visit: http://localhost:3000/open-source/spectacle/
 Both of these steps are important for validating that both the `basePath` used by the static HTML output and the `basename` used
 by the client-side router are working as expected. This is also where you'll want to validate that there are no hardcoded, inlined, or malformed asset paths that worked locally but will not resolve correctly in production!
 
+## Deployment
+
+_Only for project administrators._
+
+Our CI is configured to deploy the production build in `dist` to `formidable.com/open-source/spectacle`. This section discusses kicking the tires locally:
+
+First, install the AWS CLI:
+
+```sh
+$ brew install awscli
+```
+
+Then, set up `aws-vault` with the AWS access and secret keys for `spectacle` CI in the `AWS IAM (formidable-com)` entry in the IC vault:
+
+```sh
+$ brew install aws-vault
```

Comment:
When going through these steps I was prompted to `brew cask install aws-vault`. Should this line be changed?

Reply:
Yep! Good catch!